applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -287,18 +300,18 @@ static void audio_datapath_drift_compensation(uint32_t frame_start_ts)
 			return;
 		}
 
+		ctrl_blk.drift_comp.ctr = 0;
+
 		int32_t err_us = err_us_calculate(ctrl_blk.previous_sdu_ref_us, frame_start_ts);
 
-		/* Use asymptotic correction with small errors */
+		/* Use asymptotic to get smaller corrections */
```

Comment:
This sentence doesn't make sense anymore, suggest changing it back to the way it was. 'Asymptotic' is an adjective

Reply:
Remove these comments

Reply:
Done